travel_platform/izi_travel_client.py

```diff
@@ -0,0 +1,306 @@
+import requests
+
+
+class BaseExceptionHandler(Exception):
+    """Base class for client exceptions."""
+    pass
+
+
+class IziTravelApiError(BaseExceptionHandler):
+    pass
+
+
+class IziTravelClient(object):
+
+    def __init__(self):
+        self.api_key = "9ba66b8d-2bdc-4674-bdc4-3cc67e2c25af"
```

Comment:
It would be great to store these settings in some configuration file.
Than using configuration parser parse these variables and use here.
It's more secure and much easier for changes.